playbooks/ansible-cloud/k8s/integration.yaml

```diff
@@ -3,6 +3,15 @@
   vars:
     ansible_zuul_src_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
   tasks:
+    - name: Install molecule and collection dependencies
```

Comment:
we should move these into the test-requirements.txt file

Reply:
this is temporary, integration tests will be removed